src/repositories/profileWifiConfigs.ts

```diff
@@ -0,0 +1,73 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2021
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+import { IDbCreator } from './interfaces/IDbCreator'
+import { IProfileWifiConfigsDb } from './interfaces/IProfileWifiConfigsDb'
+import { ProfileWifiConfigs } from '../RCS.Config'
+import { mapToProfileWifiConfigs } from './mapToProfileWifiConfigs'
+import { API_UNEXPECTED_EXCEPTION } from '../utils/constants'
+import Logger from '../Logger'
+import { RPSError } from '../utils/RPSError'
+import * as format from 'pg-format'
+
+export class ProfilesWifiConfigsDb implements IProfileWifiConfigsDb {
+  db: any
+  log: Logger
+  constructor (dbCreator: IDbCreator) {
+    this.db = dbCreator.getDb()
+    this.log = new Logger('ProfilesDb')
+  }
+
+  /**
+   * @description Get AMT Profile associated wifi configs
+   * @param {string} profileName
+   * @returns {ProfileWifiConfigs[]} Return an array of wifi configs
+   */
+  async getProfileWifiConfigs (profileName: string): Promise<ProfileWifiConfigs[]> {
+    const results = await this.db.query('SELECT priority, wireless_profile_name FROM profiles_wirelessconfigs WHERE profile_name = $1', [profileName])
+    return await Promise.all(results.rows.map(async p => {
+      const result = mapToProfileWifiConfigs(p)
+      return result
+    }))
+  }
+
+  /**
+   * @description Insert wifi configs associated with AMT profile
+   * @param {ProfileWifiConfigs[]} wifiConfigs
+   * @param {string} profileName
+   * @returns {ProfileWifiConfigs[]} Return an array of wifi configs
+   */
+  async createProfileWifiConfigs (wifiConfigs: ProfileWifiConfigs[], profileName: string): Promise<boolean> {
+    try {
+      // Preparing data for inserting multiple rows
+      const wifiConfigsarray = []
+      wifiConfigs.forEach((config) => {
+        wifiConfigsarray.push([config.profileName, profileName, config.priority])
+      })
```

Comment:
what about: 
``` suggestion
const configs = wifiConfigs.map((config)=> [config.profileName, profileName, config.priority])
```

small note: try to keep type names out of variable names -- it's redundant with typescript. (ie. wifiConfigsArray = [])